Lesson31/Firat/my-app/src/components/ListItem/styles.css

```diff
@@ -1,11 +1,19 @@
 li {
-  padding: 10px;
+  padding: 10px 0px;
   border-bottom: 1px solid #ddd;
   display: flex;
   justify-content: space-between;
   align-items: center;
   background-color: #fff;
 }
+.deletebtn {
+  margin-left: 50px;
+  padding: 5px 10px;
+}
+.deletebtn:hover {
+  border: 1px solid grey;
+  border-radius: 5px;
```

Comment:
The button behaves weirdly when there's a border-radius for "hover state only. I'd recommend to add it to the main state for the button (to the .deletebtn selector). Same goes to the border properties.

For the hover state, you can change color or/and background color instead.